jobs/gtfs-schedule-validator/Dockerfile

```diff
@@ -28,6 +28,10 @@ COPY ./gtfs-validator-4.1.0-cli.jar ${V4_1_VALIDATOR_JAR}
 ENV V4_2_VALIDATOR_JAR=/gtfs-validator-4.2.0-cli.jar
 COPY ./gtfs-validator-4.2.0-cli.jar ${V4_2_VALIDATOR_JAR}
 
+# v5.0.0 from https://github.com/MobilityData/gtfs-validator/releases/download/v5.0.0/gtfs-validator-5.0.0-cli.jar
```

Comment:
Nitpick: for consistency with the other comments above that refer to major versions, we could either start this comment with "v5" or change the other ones to read "v2.0.0", "v4.0.0", etc.